### DISCUSSION TITLE
setting the pimcore portal engine

### DISCUSSION THREAD

**Discussion by Yuki1602** [pimcore/pimcore#10797] [Q&A / Support] [answered]
I am setting up pimcore portal engine and have an enterprise subscription and I want to know where to set up pimcore_portal_engine.elasticsearch.host and the index prefix as it doesn't install the extension can you tell me where do you set these 2 parameters

**Comment by fashxp** [comment, marked as answer]
It is a standard symfony parameter ... So in any loaded symfony configuration file, e.g. in your config.yaml in the parameters section.
